plutus-ledger-api/src/PlutusLedgerApi/Common/SerialisedScript.hs

```diff
@@ -59,6 +61,14 @@ implies that it is (almost certainly) an encoded script and the script does not
 
 Note: Parameterized over the ledger-plutus-version since the builtins allowed (during decoding) differs.
 -}
```

Comment:
does it make sense to name the function `assertDeserialisationSuccess` or something like that?

Reply:
Maybe, but I don't want to change the name too much because of the existing users of the API